carma-messenger-core/message/.gradle/2.14.1/taskArtifacts/cache.properties

```diff
@@ -0,0 +1 @@
+#Fri Jul 17 18:34:43 UTC 2020
```

Comment:
You commited a bug of build artifacts and binaries. Please remove them from the commit